wee_slack.py

```diff
@@ -427,7 +427,7 @@ def attach_buffer(self):
         if channel_buffer != main_weechat_buffer:
             self.channel_buffer = channel_buffer
             w.buffer_set(self.channel_buffer, "localvar_set_nick", self.server.nick)
-#            w.buffer_set(self.channel_buffer, "highlight_words", self.server.nick)
+            w.buffer_set(self.channel_buffer, "highlight_words", self.server.nick)
```

Comment:
i'm not 100% sure why, but, leaving this commented resulted in the notifications not working. so i've left it uncommented.